owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -154,16 +152,8 @@ public void onCreate(Bundle savedInstanceState) {
                     setContentDescription(getString(R.string.actionbar_settings));
         }
 
-        // Load package info
-        String temp;
-        try {
-            PackageInfo pkg = getPackageManager().getPackageInfo(getPackageName(), 0);
-            temp = pkg.versionName;
-        } catch (NameNotFoundException e) {
-            temp = "";
-            Timber.e(e, "Error while showing about dialog");
-        }
-        final String appVersion = temp + " " + BuildConfig.BUILD_TYPE + " " + BuildConfig.COMMIT_SHA1;
+        final String appVersion =
+                BuildConfig.VERSION_NAME + " " + BuildConfig.BUILD_TYPE + " " + BuildConfig.COMMIT_SHA1 + " " + BuildConfig.VERSION_CODE;
```

Comment:
VERSION_CODE would not be needed since we know it with VERSION_NAME. Just keeping the previous behaviour would be enough for the moment.

Reply:
I see it as an necessary info, but I changed it